pocean/utils.py

```diff
@@ -312,7 +312,7 @@ def get_ncdata_from_series(series, ncvar, fillna=True):
         units = getattr(ncvar, 'units', CFDataset.default_time_unit)
         calendar = getattr(ncvar, 'calendar', 'standard')
         nums = date2num(series.tolist(), units=units, calendar=calendar)
-        return np.ma.masked_invalid(nums)
+        return np.ma.masked_invalid(nums.astype(float))
```

Comment:
We could also assume that our times are always integers and change `time = nc.createVariable(axes.t, 'f8', (daxes.t,))` to `time = nc.createVariable(axes.t, 'i8', (daxes.t,))` but I have to confess I'm not sure that is a better solution.

@kwilcox any suggestions?